Faust.cpp

```diff
@@ -168,6 +166,12 @@ std::string getPathToFaustLibraries() {
     delete[] char_shareFaustDir;
     return p;
 #elif __APPLE__
+#ifdef NOBUNDLE
+    const char* myDLLPath = getMyDLLPath();
+    std::filesystem::path p = std::filesystem::path(myDLLPath).parent_path().parent_path() / "faust" ;
+    std::cerr << "[Faust]: using library path '" << p << "'" << std::endl;
+    return p.string();
+#else
```

Comment:
Can you remind me why this was added?

Reply:
this is to bring Faust.cpp in parity with its inclusion in the chuck-max distribution (which I believe builds a single bundle-less Faust.chug)